### PR TITLE
added script example for getNumericDate and examples for cfdefaultcase

### PR DESCRIPTION
**Pull request by pmturner**

### CODE REVIEW

**Comment by shaedrich** [comment]
This is confusing. Did you create a pull request for every new file plus all the files from the previous pull request? oO

**Comment by shaedrich** [comment]
That is what I expected.

So they can be closed.

File: data/en/getnumericdate.json [hunk 1]
```diff
@@ -11,6 +11,13 @@
 	"engines": {
 		"lucee": {"minimum_version": "", "notes": "", "docs": "http://docs.lucee.org/reference/functions/getnumericdate.html"}
 	},
-	"examples": [],
+	"examples": [
+		{
+			"title": "Convert a date string to a numeric date",
+			"description": "This numeric date represents the number of days after January 1, 2008.",
+			"code": "someDate = '2018-01-01';\nnumericDate = getNumericDate(someDate);\nwriteDump(numericDate);",
```
Comment (pfreitag) [inline data/en/getnumericdate.json]: This example can be condensed to simply `getNumericDate('2018-01-01')`

**Comment by pfreitag** [comment]
I merged this, but if you would like to create another PR with the simplified code example that would be great. When it detects a simple example it will automatically wrap it with `<cfoutput>#` _your example_ `#</cfoutput>`

**Pull request merged**